packages/collector/src/tracing/instrumentation/process/childProcess.js

```diff
@@ -99,15 +99,17 @@ function shimFork(original) {
           message.job &&
           typeof message.job === 'object' &&
           message.job.opts &&
-          typeof message.job.opts === 'object' &&
           entrySpan &&
           entrySpan.k === 1 &&
           processIdentityProvider
         ) {
+          if (typeof message.job.opts !== 'object') {
```

Comment:
@willianpc `typeof null` is `object`. So if for some reason we get `message.job.opts = null` (unlikely, I know) this will result in an NPE a few lines later. I'd rather write this as 
```
if (message.job.opts == null) {
 message.job.opts = {};
}
```

Reply:
good catch. will do